CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -362,6 +380,10 @@ public struct CalculateCalendarLogic {
             case (1966...1999, 10, 11, .mon):
                 return true
             
+            //2019年10月22日： 2019年だけ即位礼正殿の儀
+            case (emperorthroneYear, 10, 22, _):
```

Comment:
__[SwiftLint]__ Case statements should vertically align with their enclosing switch statement.

<sub>posted by Sider</sub>